components/infobox/extensions/commons/placement_stats.lua

```diff
@@ -0,0 +1,133 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:InfoboxPlacementStats
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local placementStats = {}
+local Class = require('Module:Class')
+local Template = require('Module:Template')
+local Table = require('Module:Table')
+local Medal = require('Module:Medal')
+local Settings = require('Module:InfoboxPlacementStats/Settings')
```

Comment:
Let's follow the NotabilityChecker here and call this

```suggestion
local Config= require('Module:InfoboxPlacementStats/config')
```

Although  I'm open to

```suggestion
local Config= require('Module:InfoboxPlacementStats/Config')
```

too